src/AppInstallerCommonCore/Public/winget/UserSettings.h

```diff
@@ -106,6 +157,8 @@ namespace AppInstaller::Settings
         }
 
         static std::filesystem::path SettingsFilePath();
+        static std::tuple<std::string, std::string, std::string> GetFeatureInfo(ExperimentalFeature feature);
```

Comment:
>std::tuple<std::string, std::string, std::string> [](start = 15, length = 49)

Don't use a tuple, take the time to create a struct with 3 names so that we can actually read the code.